packages/runtime/src/widgets/Form/FormItem.tsx

```diff
@@ -17,9 +17,11 @@ export const EnsembleFormItem: React.FC<EnsembleFormItemProps<unknown>> = (
 ) => {
   const { values, ...rest } = props;
   const { backgroundColor: _, ...formItemStyles } = values?.styles ?? {};
+
   return (
     <AntForm.Item
       className={values?.styles?.names}
+      id={values?.id ?? values?.label}
```

Comment:
I don't think this is the issue...can you create a kitchen sink example?

Reply:
I added a example in kitchen-sink for this, where you can check this

Reply:
Hmmm can you explain to me what the issue was and how this resolves this?

Reply:
@evshi https://ant.design/components/input
according to document, when we wrap any form elements with the form item, then form item take all the control of the form element, and the value, and id prop of the form elements are controlled by the form item only.

Reply:
@sagardspeed2 but your change (removing `name` control) caused Form not to be submitted anymore.

Reply:
@sagardspeed2 please explain why the issue occurs not just point at documentation. Only if you understand the root cause can you create a proper fix.

Reply:
I removed this since it's not necessary for the solution to work fyi @sagardspeed2